src/packages/excalidraw/package.json

```diff
@@ -75,10 +75,11 @@
   "scripts": {
     "gen:types": "tsc --project ../../../tsconfig-types.json",
     "build:umd": "rm -rf dist && cross-env NODE_ENV=production webpack --config webpack.prod.config.js && cross-env NODE_ENV=development webpack --config webpack.dev.config.js && yarn gen:types",
+    "build:umd:dev": "cross-env NODE_ENV=development webpack --config webpack.dev.config.js",
```

Comment:
is this one needed ? coz we already have a command which uses dev build when using example `yarn build:example`

Reply:
Removed in eb76191.